fury/tests/test_colormap.py

```diff
@@ -115,3 +115,10 @@ def test_lab2rgb():
     res = np.round(colormap._lab2rgb(lab_color))
     res[res < 0] = 0
     npt.assert_array_almost_equal(res, expected)
+
+
+def test_hex_to_rgb():
+    expected = np.array([0.99609375, 0.99609375, 0.99609375])
```

Comment:
These values are weird and wrong, it should be 1 as you did before `np.array([1, 1, 1])`

Reply:
DOne.